ajax/libs/mapbox-gl/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "mapbox-gl",
+  "description": "A WebGL interactive maps library",
+  "version": "0.32.1",
+  "main": "src/index.js",
+  "filename": "src/index.min.js",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/mapbox/mapbox-gl-js.git"
+  }, 
+  "npmName": "mapbox-gl",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "dist/mapbox-gl.css",
+        "src/index.js"
+      ]
```

Comment:
I found that there is no `src` directory in NPM package, so I think we can just set `dist` as basePath.